client/components/select-dropdown/index.jsx

```diff
@@ -219,6 +236,7 @@ class SelectDropdown extends Component {
 		const dropdownClassName = classNames( dropdownClasses );
 
 		const selectedText = this.getSelectedText();
+		const selectedIcon = this.getSelectedIcon();
```

Comment:
@mtias are those supposed to be parens or are you asking me to create an array called `getSelectedItem`? I'm unfamiliar with this syntax.